flight/uavobjectlib/Makefile

```diff
@@ -66,8 +66,15 @@ LDFLAGS += -Wl,--fatal-warnings
 SRC += $(filter-out $(OPUAVSYNTHDIR)/uavobjectsinit.c,$(wildcard $(OPUAVSYNTHDIR)/*.c))
 NO_AUTO_UAVO = YES
 
+# Backup the environmental variable before entering firmware-common.mk
```

Comment:
No, I'm saying why do you restore the original value?

This makefile doesn't need it put back, and no other makefiles see the consequence of you putting it back.  Hence, this is vestigial :/